test/unit/httpBinding-test.js

```diff
@@ -1,694 +0,0 @@
-
-            contextBrokerUnprovMock
-                .matchHeader('fiware-service', 'TestService')
-                .matchHeader('fiware-servicepath', '/testingPath')
-                .post('/v1/updateContext', utils.readExampleFile('./test/contextRequests/unprovisionedMeasure.json'))
-                .reply(200, utils.readExampleFile('./test/contextResponses/unprovisionedSuccess.json'));
-
-            request(groupCreation, function (error, response, body) {
-                done();
-            });
-        });
-
-        it('should end up with a 200OK status code', function (done) {
-            request(getOptions, function (error, response, body) {
-                should.not.exist(error);
-                response.statusCode.should.equal(200);
-                done();
-            });
-        });
-        it('should send a new update context request to the Context Broker with just that attribute', function (done) {
-            request(getOptions, function (error, response, body) {
-                contextBrokerUnprovMock.done();
-                done();
-            });
-        });
-        it('should add a protocol to the registered devices', function (done) {
-            const getDeviceOptions = {
-                url: 'http://localhost:4061/iot/devices/MQTT_UNPROVISIONED',
-                method: 'GET',
-                headers: {
-                    'fiware-service': 'TestService',
-                    'fiware-servicepath': '/testingPath'
-                }
-            };
-
-            request(getOptions, function (error, response, body) {
-                request(getDeviceOptions, function (error, response, body) {
-                    should.not.exist(error);
-
-                    const parsedBody = JSON.parse(body);
-
-                    response.statusCode.should.equal(200);
-                    should.exist(parsedBody.protocol);
-                    parsedBody.protocol.should.equal('MQTT_UL');
-                    done();
-                });
-            });
-        });
-        it('should add a transport to the registered devices', function (done) {
-            const getDeviceOptions = {
-                url: 'http://localhost:' + config.iota.server.port + '/iot/devices/MQTT_UNPROVISIONED',
-                method: 'GET',
-                headers: {
-                    'fiware-service': 'TestService',
-                    'fiware-servicepath': '/testingPath'
-                }
-            };
-
-            request(getOptions, function (error, response, body) {
-                request(getDeviceOptions, function (error, response, body) {
-                    should.not.exist(error);
-
-                    const parsedBody = JSON.parse(body);
-
-                    response.statusCode.should.equal(200);
-                    should.exist(parsedBody.transport);
-                    parsedBody.transport.should.equal('HTTP');
-                    done();
-                });
-            });
-        });
```

Comment:
This it section is not present on NGSI-v2 test [here](https://github.com/telefonicaid/iotagent-ul/blob/c71f1d3cb4cd4c81c51446d3f0e16c020ae1dae0/test/unit/ngsiv2/httpBindings-test.js#L169-L251)